lightning/backend/session.go

```diff
@@ -94,9 +101,10 @@ func (t *transaction) SetAssertion(kv.Key, kv.AssertionType) {}
 // transaction type and provides the session variables to the TiDB library
 // optimized for Lightning.
 type session struct {
-	sessionctx.Context
```

Comment:
keep this, and remove all methods not called, so that we'll know what is the proper implementation when they are actually called.

Reply:
We have called `Context.SetValue` to reuse the `CommonAddRecordCtx`.  If we keep this, the Context is nil, thus cause panic with nil pointer.